### ISSUE TITLE
Cannot work

### ISSUE THREAD

**Issue by shiuduy** [undefiorg/vanibnb#1]
hi so when i started to install this with go get -u github.com/undefiorg/vanibnb i got this error over and over
go: downloading github.com/undefiorg/vanibnb v0.0.0-20210424043445-163d671d03c0
go get: github.com/undefiorg/vanibnb@v0.0.0-20210424043445-163d671d03c0: parsing go.mod:
        module declares its path as: undefiorg/vanibnb
                but was required as: github.com/undefiorg/vanibnb

**Comment by katopz** [comment]
just update `go.mod` not sure if it work for you.

**Comment by shiuduy** [comment]
yeah i did i removed the https:// and still same error can you please tell me another way to run the program manually , i would really appreciate it thank you very much for answering quickly

**Comment by shiuduy** [comment]
actually it did work now but i faced another error which is
 cannot find module providing package github.com/tendermint/tender

**Comment by shiuduy** [comment]
i guess because the github page is not existent anymore , do you have any alternative ?

**Comment by katopz** [comment]
not at the moment, i will get back to this when i have some free time to burn, feel free to PR if you found a fix.

hint: you can take a look at other forked. i think someone already have a fix already.
